core/src/test/java/org/elasticsearch/index/IndexModuleTests.java

```diff
@@ -315,7 +327,8 @@ public void testDefaultQueryCacheImplIsSelected() throws IOException {
                 .put("path.home", createTempDir().toString())
                 .put(IndexMetaData.SETTING_VERSION_CREATED, Version.CURRENT).build();
         IndexModule module = new IndexModule(IndexSettingsModule.newIndexSettings(new Index("foo"), indexSettings), null, new AnalysisRegistry(null, environment));
-        IndexService indexService = module.newIndexService(nodeEnvironment, deleter, nodeServicesProvider, mapperRegistry);
+        IndexService indexService = module.newIndexService(nodeEnvironment, deleter, nodeServicesProvider, mapperRegistry, shardId -> {
+        });
```

Comment:
Nit: wrap to the preceding line (and on lines 141, 155, 180, 211, 239, 257, 272 and 320).